components/camel-paho/pom.xml

```diff
@@ -48,15 +48,17 @@
 
         <dependency>
             <groupId>org.apache.camel</groupId>
-            <artifactId>camel-test-junit5</artifactId>
+            <artifactId>camel-core-catalog</artifactId>
```

Comment:
I wonder why you need the catalog ?

Reply:
TBH, I don't know. It was [there previously](https://github.com/apache/camel/blob/main/components/camel-paho/pom.xml#L54-L58). It was just displayed here due to the text block movement.

Reply:
Looking more closely at this one, it seems it was added as part of [CAMEL-14499](https://issues.apache.org/jira/browse/CAMEL-14499). However, it appears to me the `camel-core-catalog` is not needed since the tests pass without it ... maybe we can clean this up (in paho and in other components on that list).

Reply:
Yes it can be removed

Reply:
Great. I'll send a separate PR cleaning this up.

Reply:
Done: #9649.